bb-email/src/test/resources/email.imap.properties

```diff
@@ -1,12 +0,0 @@
-email.username=qatest_user
```

Comment:
When managing multiple mailboxes it would be easier for me to have separate properties file for every mailbox.
Then if keys didn't contain `${id}` (id could be a filename without extension) then i could easily compare those files (when for example the difference should be only in username/password.
* less changes in this pull request
* configuration id easy to change (one change instead of change in every key)
* creating a new configuration easy (just copy a file)

What do you think?